features/txt-parfmt-props.feature

```diff
@@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
   I need a ParagraphFormat object with read/write formatting properties
 
 
+  @wip
+  Scenario Outline: Get tab stops object
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops is a CT_TabStops object
+      
+    Examples:  values for paragraph_format.tab_stop elements 
+      | value |
+      | no    |
+      | some  |
+      
```

Comment:
> In features/txt-parfmt-props.feature 
> <https://github.com/python-openxml/python-docx/pull/261#discussion_r56744916>:
>
> > @@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
> >    I need a ParagraphFormat object with read/write formatting properties
> >
> >
> > +  @wip
> > +  Scenario Outline: Get tab stops object
> > +    Given a paragraph having <value> tab_stop elements
> > +     Then paragraph_format.tab_stops is a CT_TabStops object
> > +
> > +    Examples:  values for paragraph_format.tab_stop elements
> > +      | value |
> > +      | no    |
> > +      | some  |
> > +
>
> Okay, this acceptance test is merged in feature/tabstops on the main 
> repo. You should rebase onto that and you can proceed with the 
> implementation of this bit. You can submit all those commits (there 
> might only be one), I expect it will be close enough for me to merge 
> with any tweaks.
>
> I'll provide feedback on the rest now, and you can submit them in 
> commits following the ParagraphFormat.tab_stops commit(s).
>

Hi Steve,

I'm back from vacation and ready to dive in, but I'm confused by what 
you mean here by "the implementation of this bit."

I thought the process was to get a fairly comprehensive set of 
acceptance tests in place, then do a series of unit test - 
implementation pairings.  Am I supposed to do acceptance test - unit 
test - implementation of each step as we go?  If not, I'm not clear what 
is needed under "implementation" for this first acceptance test.  It 
looks like it fails at the right spot pre-implementation to me, so I 
must be missing something.

David

-- 
David K. Woods, Ph.D.
Researcher, Lead Transana Developer
Wisconsin Center for Education Research
University of Wisconsin, Madison
http://www.transana.org